components/kyma-environment-broker/internal/process/upgrade_kyma/get_kubeconfig_step.go

```diff
@@ -33,6 +33,7 @@ func (s *GetKubeconfigStep) Name() string {
 
 func (s *GetKubeconfigStep) Run(operation internal.UpgradeKymaOperation, log logrus.FieldLogger) (internal.UpgradeKymaOperation, time.Duration, error) {
 	if operation.Kubeconfig != "" {
+		operation.InputCreator.SetKubeconfig(operation.Kubeconfig)
```

Comment:
input creator is not stored in the database in case of requeue, need to take it from the operation's kubeconfig that is stored